Cache/Hashing.lean

```diff
@@ -11,10 +11,26 @@ namespace Cache.Hashing
 
 open System IO
 
+/--
+The `HashMemo` contains all information `Cache` needs about the modules:
+* the name
+* its imports
+* the file's hash (in `hashMap` and `cache`)
+
+additionally, it contains the `rootHash` which reflects changes to Mathlib's
+Lake project settings.
+-/
 structure HashMemo where
+  /-- Hash of mathlib's lake project settings. -/
   rootHash : UInt64
+  /-- Stores the imports of a module -/
```

Comment:
Is this `.lean` -> `.olean`, `.lean` -> `.lean`, or `.olean` -> `.olean`?

Reply:
.lean -> .lean

updated the docstring